app/realtime.php

```diff
@@ -205,6 +208,21 @@
         Realtime::subscribe($project->getId(), $connection, $roles, $subscriptions, $connections, $channels);
 
         $server->push($connection, json_encode($channels));
+
```

Comment:
We should probably return the connection to the pull on an exception as well. What about the HTTP server do we want to try and fully leverage the connection pool for both APIs?

Reply:
Definitely should be in the exception 👍🏻 had it like this for testing.

I still not 100% understand the behavior behind the Connection Pools. Will make this part of the Issue I will raise over at the Swoole repo over the weekend 🙂